cap/modules/mail/utils.py

```diff
@@ -50,6 +50,19 @@ def get_cms_stat_recipients(record, config):
     if pdf_mail and record.get('parton_distribution_functions', None):
         recipients.append(pdf_mail)
 
+    # mail for ML surveys
```

Comment:
let write a test for this one (same test can be for the `PDF` field also).. be careful to supress email sending